INSTALL.rst

```diff
@@ -0,0 +1,363 @@
+.. The source of this document is INSTALL. During the doc build process,
+.. this file is copied over to doc/users/installing.rst.
+.. Therefore, you must edit INSTALL, *not* doc/users/installing.rst!
+.. _pip: https://pypi.python.org/pypi/pip/
+
+==========
+Installing
+==========
+
+.. note::
+
+    If you wish to contribute to the project, it's recommended you
+    :ref:`install the latest development version<install_from_source>`.
+
+
+
+.. contents.. 
+
+Installing an official release
+==============================
+
+Matplotlib and most of its dependencies all available as wheel packages for
+OSX, windows and many linux distributions::
+
+  pip install -U matplotlib
+
+Third-party distributions of Matplotlib
+=======================================
+
+Scientific Python distributions: Conda, Canopy...
+--------------------------------------------------
+
+The first option is to use one of the pre-packaged Python distributions that
+already provide Matplotlib built-in.  Both `Anaconda
+<https://www.continuum.io/downloads/>`_ and `Canopy
+<https://www.enthought.com/products/canopy/>`_ are both excellent choices that
+"just work" out of the box for Windows, OSX and common Linux platforms.  Both
+of these distributions include Matplotlib and *lots* of other useful tools.
+
+Linux : using your package manager
+----------------------------------
+
+If you are on Linux, you might prefer to use your package manager.  Matplotlib
+is packaged for almost every major Linux distribution.
+
+* Debian / Ubuntu : ``sudo apt-get install python-matplotlib``
+* Fedora / Redhat : ``sudo yum install python-matplotlib``
```

Comment:
Red Hat.

on Arch, `sudo pacman -S python-matplotlib` (if you care to add it)

Depending on the distro it may be either python-matplotlib/python2-matplotlib (e.g. Fedora, Arch) or python-matplotlib/python3-matplotlib.  Check at https://pkgs.org/download/matplotlib.